docs/docs/adding-tags-and-categories-to-blog-posts.md

```diff
@@ -48,25 +48,16 @@ Try running the following query in Graph<em>i</em>QL (`localhost:8000/___graphql
 
 ```graphql
 {
-  allMarkdownRemark(
-    sort: { order: DESC, fields: [frontmatter___date] }
-    limit: 1000
-  ) {
-    edges {
-      node {
-        fields {
-          slug
-        }
-        frontmatter {
-          tags
-        }
-      }
+  allMarkdownRemark {
+    group(field: frontmatter___tags) {
+      tag: fieldValue
+      totalCount
     }
   }
 }
 ```
 
-The resulting data includes the `slug` field and `tags` frontmatter for each post, which is all the data we'll need to create pages for each tag which contain a list of posts under that tag. Let's make the tag page template now:
+The above query groups posts by `tags`, and return each `tag` with the number of posts for each as `totalCount`. We could also extract some post data in each group, but to create a page for each tag, we only need to know the tag name. Let's make the tag page template now:
```

Comment:
The sentence has a lot of small words and makes it hard to read in my opinion.

I'm suggesting:
```suggestion
The above query groups posts by `tags`, and returns each `tag` with the number of posts as `totalCount`. As an addition, we could extract some post data in each group if we need to. To keep this tutorial small, we're only using the tag name in our tag pages. Let's make the tag page template now:
```